packages/crayons-core/src/components/radio/radio.tsx

```diff
@@ -100,25 +103,28 @@ export class Radio {
       <Host
         class='radio-container'
         onClick={() => this.toggle()}
-        role='radio'
-        tabIndex='0'
+        role="radio"
+        tabIndex="-1"
+        aria-label={this.label}
+        aria-describedby="description"
         aria-disabled={this.disabled ? 'true' : 'false'}
-        aria-checked={this.checked}
+        aria-checked={`${this.checked}`}
```

Comment:
Is this being done to force-convert `this.checked` into a string?

Reply:
Correct Manoj, this is done to convert to string. Have changed to `this.checked ? 'true': 'false'` to be more clear